sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -771,12 +771,55 @@ public DigitalTwinsResponse<Void> updateComponentWithResponse(String digitalTwin
 </details>
 
 ## Query
-<details>
-<summary><b>APIs</b></summary>
+<details><summary><b>Async APIs</b></summary>
+
+  ```java
+  /**
+  * Query digital twins.
+  *
+  * @param query The query string, in SQL-like syntax.
+  * @return A {@link PagedFlux} of application/json for the query result items.
+  */
+  @ServiceMethod(returns = ReturnType.COLLECTION)
+  public PagedFlux<String> query(String query)
```

Comment:
should the naming be queryTwins? I know in dotnet it was queryAsync but thinking since it may be more meaningful

Reply:
Parity is probably more important here. Feel free to bring it up in a meeting and we can all discuss collectively.